CHANGELOG.md

```diff
@@ -2,4 +2,13 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
```

Comment:
@Jefiozie it shows the changelog file as a change in the PR. Is this normal? I think it should not 🤔

Reply:
That is a great question, I'm not sure why it is here. Let me rebsae it once more.